client/necsus.js

```diff
@@ -219,9 +219,26 @@ let app = new Vue({
       return null
     },
     addBot: function() {
+      // GH-style names w. animals, eg. Owl-Cod-Otter
+      const animals = ["Aardvark", "Alligator", "Alpaca", "Anaconda", "Ant", "Anteater", "Antelope", "Aphid",
+                       "Armadillo", "Asp", "Baboon", "Badger", "Barracuda", "Bass", "Bat", "Beaver", "Bedbug",
+                       "Bee", "Bee-eater", "Bird", "Bison", "Bobcat", "Buffalo", "Butterfly", "Buzzard",
+                       "Camel", "Carp", "Cat", "Caterpillar", "Catfish", "Cheetah", "Chicken", "Chimpanzee",
+                       "Chipmunk", "Cobra", "Cod", "Condor", "Cougar", "Cow", "Coyote", "Crab", "Cricket",
+                       "Crocodile", "Crow", "Cuckoo", "Deer", "Dinosaur", "Dog", "Dolphin", "Donkey", "Dove",
+                       "Dragonfly", "Duck", "Eagle", "Eel", "Elephant", "Emu", "Falcon", "Ferret", "Finch",
+                       "Fish", "Flamingo", "Flea", "Fly", "Fox", "Frog", "Goat", "Goose", "Gopher", "Gorilla",
+                       "Hamster", "Hare", "Hawk", "Hippopotamus", "Horse", "Hummingbird", "Husky", "Iguana",
+                       "Impala", "Kangaroo", "Lemur", "Leopard", "Lion", "Lizard", "Llama", "Lobster", "Margay",
+                       "Monkey", "Moose", "Mosquito", "Moth", "Mouse", "Mule", "Octopus", "Orca", "Ostrich",
+                       "Otter", "Owl", "Ox", "Oyster", "Panda", "Parrot", "Peacock", "Pelican", "Penguin",
+                       "Perch", "Pheasant", "Pig", "Pigeon", "Porcupine", "Quagga", "Rabbit", "Raccoon", "Rat",
+                       "Rattlesnake", "Rooster", "Seal", "Sheep", "Skunk", "Sloth", "Snail", "Snake", "Spider",
+                       "Tiger", "Whale", "Wolf", "Wombat", "Zebra"]
+      const shuffled = animals.sort(() => 0.5 - Math.random());
```

Comment:
Sorting using a random comparator is not quite the same thing as shuffling, and doesn't make much sense (the sorting algorithm could ask if `x <= y` and get `true`, and later ask the same question and get `false`). How you can get a random shuffle via sorting is by generating a random number for each element of the list, hence turning every item into a pair of `(data, random number)`, and sorting by the random number, like in this answer: <https://stackoverflow.com/a/46545530>